scripts/releaseSite.sc

```diff
@@ -0,0 +1,114 @@
+import $ivy.`com.lihaoyi::requests:0.6.5`
+import $ivy.`com.lihaoyi::upickle:0.9.5`
+
+// format: off
+
+// Assumes mdoc has been run.
+@main
+def main(): Unit = {
+
+  if (!os.exists(os.pwd / "modules" / "docs" / "target" / "mdoc"))
+    sys.error("Have you run mdoc ?")
+
+  val website = os.pwd / "website"
+
+  website.yarn("install")
+  // Freezing version
+
+  val version     = sys.env("DRONE_TAG").dropWhile(_ == 'v')
+  val siteConfig  = ujson.read(os.read(website / "siteConfig.json"))
+  val orgName     = siteConfig("organizationName").str
+  val projectName = siteConfig("projectName").str
+  val redirectUrl = projectName + "/index.html"
+  val html        = redirectHtml(redirectUrl)
+
+  val currentBranch = git("rev-parse", "--abbrev-ref", "HEAD")
+  val currentCommit = git("rev-parse", "HEAD")
+
+  val siteBranch        = "gh-pages"
+  val frozenDocsBranch  = "frozen-docs"
```

Comment:
I went ahead and created this branch manually (and added it to the list of protected branches, in order to prevent its removal)